ViewLayer/DebugDisplay/MotorFaultPage/MotorFaultView/MotorFaultView.cpp

```diff
@@ -0,0 +1,370 @@
+    initializeLabel(highVoltageIsolationFault_, layoutB, ERRORSTYLESHEET, current_widthB);
+    initializeLabel(internalCommununicationFault_, layoutB, ERRORSTYLESHEET, current_widthB);
+    initializeLabel(internalConversionFault_, layoutB, ERRORSTYLESHEET, current_widthB);
+    initializeLabel(internalLogicFault_, layoutB, ERRORSTYLESHEET, current_widthB);
+    initializeLabel(internalMemoryFault_, layoutB, ERRORSTYLESHEET, current_widthB);
+    initializeLabel(internalThermistorFault_, layoutB, ERRORSTYLESHEET, current_widthB);
+    initializeLabel(lowCellVoltageFault_, layoutB, ERRORSTYLESHEET, current_widthB);
+    initializeLabel(openWiringFault_, layoutB, ERRORSTYLESHEET, current_widthB);
+    initializeLabel(packVoltageSensorFault_, layoutB, ERRORSTYLESHEET, current_widthB);
+    initializeLabel(powerSupplyFault12V_, layoutB, ERRORSTYLESHEET, current_widthB);
+    initializeLabel(thermistorFault_, layoutB, ERRORSTYLESHEET, current_widthB);
+    initializeLabel(voltageRedundancyFault_, layoutB, ERRORSTYLESHEET, current_widthB);
+    initializeLabel(weakCellFault_, layoutB, ERRORSTYLESHEET, current_widthB);
+    initializeLabel(weakPackFault_, layoutB, ERRORSTYLESHEET, current_widthB);
+
+    initializeLabel(cclReducedDueToAlternateCurrentLimit_, layoutB, LIMITSTYLESHEET, current_widthB);
+    initializeLabel(cclReducedDueToChargerLatch_, layoutB, LIMITSTYLESHEET, current_widthB);
+    initializeLabel(cclReducedDueToHighCellResistance_, layoutB, LIMITSTYLESHEET, current_widthB);
+    initializeLabel(cclReducedDueToHighCellVoltage_, layoutB, LIMITSTYLESHEET, current_widthB);
+    initializeLabel(cclReducedDueToHighPackVoltage_, layoutB, LIMITSTYLESHEET, current_widthB);
+    initializeLabel(cclReducedDueToHighSoc_, layoutB, LIMITSTYLESHEET, current_widthB);
+    initializeLabel(cclReducedDueToTemperature_, layoutB, LIMITSTYLESHEET, current_widthB);
+    initializeLabel(dclandCclReducedDueToCommunicationFailsafe_, layoutB, LIMITSTYLESHEET, current_widthB);
+    initializeLabel(dclandCclReducedDueToVoltageFailsafe_, layoutB, LIMITSTYLESHEET, current_widthB);
+    initializeLabel(dclReducedDueToHighCellResistance_, layoutB, LIMITSTYLESHEET, current_widthB);
+    initializeLabel(dclReducedDueToLowCellVoltage_, layoutB, LIMITSTYLESHEET, current_widthB);
+    initializeLabel(dclReducedDueToLowPackVoltage_, layoutB, LIMITSTYLESHEET, current_widthB);
+    initializeLabel(dclReducedDueToLowSoc_, layoutB, LIMITSTYLESHEET, current_widthB);
+    initializeLabel(dclReducedDueToTemperature_, layoutB, LIMITSTYLESHEET, current_widthB);
+
+    ui_.motor0ContentsWidget().setLayout(layout0);
+    ui_.motor1ContentsWidget().setLayout(layout1);
+    ui_.batteryContentsWidget().setLayout(layoutB);
+
+    connectMotorFaults(motorFaultsPresenter_);
+    connectBatteryFaults(batteryFaultsPresenter_);
+}
+
+MotorFaultView::~MotorFaultView()
+{
+}
+
+void MotorFaultView::initializeLabel(QLabel& label, QLayout*& layout, QString& styleSheet, int width)
```

Comment:
Since the width isn't changing anyways, let's use `layout.size().width()` instead of having an argument.